go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/codingpop/simplerest
+
+go 1.15
+
+require github.com/go-chi/chi v4.1.2+incompatible
```

Comment:
@johanbrandhorst I couldn't take my mind off this. Why does this say incompatible? Does it mean it has some incompatibilities I am not aware of?

Reply:
Great question! Here's an answer on the go modules wiki: https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-v2-package-that-has-not-opted-into-modules-what-does-incompatible-mean. Basically, it means that the module is >= v2, but not using a /v2 suffix in the package path. It doesn't mean that it's dangerous to use, just that it doesn't fulfill all the guarantees a proper go module can give around compatibility. Nothing really to worry about most of the time.

Reply:
Thanks for the explanation